accounts/ledger_account.md

```diff
@@ -0,0 +1,21 @@
+## LedgerAccount
+
+```
+// should implement the `IAccount` interface
```

Comment:
Just a suggestion, when we are saying things like: "should implement: xInterface" - could we make maybe "xInterface" as a link to where it's defined?